test/expected.txt

```diff
@@ -184,8 +184,9 @@
 || 0 persons found!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
+|| 	1. Betsy Choo  Phone Number: 222222  Email: benchoo@nus.edu.sg
```

Comment:
Glad to see that you ran tests when you implemented/extended a feature.
Shouldn't the user guide be updated too? Recall from our tutorial.